packages/components/psammead-figure/src/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Figure should render correctly 1`] = `
+.c0 {
+  margin: 0;
+  width: 100%;
```

Comment:
The snapshot doesn't seem to include the `padding-bottom` value.

Reply:
@sareh - correct. It looks I was importing the wrong `GEL_SPACING_DBL` variable. It's updated now.